lib/tasks/fake_data.rake

```diff
@@ -216,11 +216,285 @@ namespace :fake_data do
     end
   end
 
+  desc 'create fake journeys with associated events'
+  task create_journeys: :environment do
+    puts 'create_journeys...'
+    Move
+        .left_outer_joins(:journeys).where(journeys: { move_id: nil })
+        .where(status: %w[completed booked requested])
+        .find_each do |move|
+      # Creates some random journeys between A and B
+      current_location = move.from_location
+      number_of_journeys = rand(1..5)
+      journey_counter = 1
+      supplier = move.from_location.suppliers.first || Supplier.all.sample # pick a supplier based on the from location or at random
+      to_location = move.to_location || Location.order('RANDOM()').where(location_type: 'prison').first # pick a random prison if to_location was not specified
```

Comment:
For moves without a `to_location` set, since we are creating journeys for them with a final destination, should we mirror that in the move and persist the `to_location` there as well?

Reply:
good question; moves without a `to_location` are a special type of police(?) to prison move, where the supplier and police know that the person is going to the default local prison. We don't have (and don't want to maintain) the list of default prisons, so we leave `to_location` blank. Removing the blank could upset some of the logic for these types of move.